File: letnik1/semester2/ois/vaje/gravitacija.py
import math

def main():
    # Constants
    c = 6.674 * math.pow(10, -11)  # Gravitational constant
    m = 5.972 * math.pow(10, 24)   # Mass of the Earth
    r = 6.371 * math.pow(10, 6)    # Radius of the Earth

    # User input
    input_height = int(input("Enter the height above the Earth's surface (in meters): "))

    # Calculate gravitational force
    output = (c * m) / math.pow(r + input_height, 2)

    # Print the result
    print(output)

if __name__ == "__main__":
    main()

File: letnik1/semester2/ois/vaje/Gravitacija.java
import java.util.*;
class Gravitacija{
    public static void main(String[] args){
        final double c = 6.674 * Math.pow(10,-11);
        final double m = 5.972 * Math.pow(10,24);
        final double r = 6.371 * Math.pow(10,6);
        Scanner sc = new Scanner(System.in);
        int input = sc.nextInt();
        double output = c*m/Math.pow(r+input,2);
        System.out.println(output);
    }
}